app/views/shared/_error.html.erb

```diff
@@ -1,18 +1,20 @@
-<div class="govuk-error-summary" id="error_explanation" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
-  <h2 class="govuk-error-summary__title" id="error-summary-title">
-    <%= t('generic.errors.problem_text') %>
-  </h2>
-  <div class="govuk-error-summary__body">
-    <ul class="govuk-list govuk-error-summary__list">
-      <% if @error.is_a? String %>
-        <li>
-          <%= link_to_accessible @error, "#correct" %>
-        </li>
-      <% else %>
-        <li>
-          <%= link_to_accessible @error.values.first, "##{@error.keys.first}" %>
-        </li>
-      <% end %>
-    </ul>
+<div class="govuk-error-summary" id="error_explanation" tabindex="-1" data-module="govuk-error-summary">
+  <div role="alert">
+    <h2 class="govuk-error-summary__title">
+      <%= t('generic.errors.problem_text') %>
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <% if @error.is_a? String %>
+          <li>
+            <%= link_to_accessible @error, "#correct" %>
+          </li>
+        <% else %>
+          <li>
+            <%= link_to_accessible @error.values.first, "##{@error.keys.first}" %>
+          </li>
+        <% end %>
```

Comment:
As the only thing that changes, based on the conditon, is the link, could this be improved, for example:

```
<li>
  <% if @error.is_a? String %>
    <%= link_to_accessible @error, "#correct" %>
  <% else %>
    <%= link_to_accessible @error.values.first, "##{@error.keys.first}" %>
  <% end %>
</li>
```